static/js/util/util.js

```diff
@@ -74,11 +76,12 @@ export function makeProfileProgressDisplay(active: number) {
 
   const elements = [];
 
-  for (let i = 0; i < numCircles; ++i) {
+  let activeTab = Object.keys(PROFILE_STEP_LABELS).findIndex(k => k === active);
```

Comment:
The keys in an object are not defined to be in any particular order, though browsers will generally keep insertion order just because: http://stackoverflow.com/questions/5525795/does-javascript-guarantee-object-property-order

We should change `PROFILE_STEP_LABELS` to be an array if we want to guarantee order